execute/plugin.go

```diff
@@ -186,7 +186,8 @@ func getPendingReportsForExecution(
 	lggr.Debugw("grouped commits before removing fully executed reports",
 		"groupedCommits", groupedCommits, "count", len(groupedCommits))
 
-	// Remove fully executed reports.
+	rangesBySelector := make(map[cciptypes.ChainSelector][]cciptypes.SeqNumRange)
+	filteredReports := make(map[cciptypes.ChainSelector][]exectypes.CommitData)
```

Comment:
```suggestion
	executableReportsPerChain := make(map[cciptypes.ChainSelector][]exectypes.CommitData)
```

Or just `executableReports` if we wanna be more compact.

Reply:
renamed to `executableReports` so it's always accessed as `executableReports[selector]` rather than `executableReportsBySelector[selector]`